examples/react-instantsearch-hooks/src/vite-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```

Comment:
do we actually need this one? I don't think we use vite-specific types

Reply:
I removed it, it only declared modules for import of various file types, that we won't use in the demo.